saleor/static/dashboard-next/components/Money/Money.tsx

```diff
@@ -11,17 +13,22 @@ export const Money: React.StatelessComponent<MoneyProps> = ({
   amount,
   currency,
   typographyProps
-}) => {
-  const money =
-    currency === "%"
-      ? [amount, currency].join(" ")
-      : amount.toLocaleString(navigator.language, {
-          currency,
-          style: "currency"
-        });
-  if (typographyProps) {
-    return <Typography {...typographyProps}>{money}</Typography>;
-  }
-  return <>{money}</>;
-};
+}) => (
+  <LocaleConsumer>
+    {locale => {
+      const money =
+        currency === "%"
```

Comment:
Yes... and no. For semantic component separation, it would be okay to have component formatting only money amounts and component formatting percentage amounts, but technically it would be almost the same component - only difference between them would be
```javascript
amount.toLocaleString(locale, { currency, style: "currency" })
```
for money amounts, and
```javascript
amount.toLocaleString(locale, { style: "percent" })
```
for percentage amounts.